packages/frontend/src/content/ShowNote.tsx

```diff
@@ -8,40 +8,40 @@ import { HomeButton, Loading, PageContainer } from "../components";
 
 const ShowNote = (props: RouteComponentProps<{ noteId: string }>) => {
   const { noteId } = props;
-  const [isLoading, setIsLoading] = useState(true);
   const [noteContent, setNoteContent] = useState("");
   const [attachment, setAttachment] = useState("");
   const [attachmentURL, setAttachmentURL] = useState("");
+  const [errorMsg, setErrorMsg] = useState("");
```

Comment:
The `errorMsg` is not set in this component, do it's not going to be displayed.
This can be removed.
```suggestion
```